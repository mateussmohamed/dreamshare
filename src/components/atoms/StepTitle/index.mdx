---
name: StepTitle
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import StepTitle from './index'

# StepTitle
<PropsTable of={StepTitle} />

<Playground>
  <StepTitle>
    SHARE YOUR HOLIDAY DREAM
  </StepTitle>
</Playground>
