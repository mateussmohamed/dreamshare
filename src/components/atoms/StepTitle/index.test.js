import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import StepTitle from './index'

describe('[<StepTitle />]', () => {
  const wrap = (props = {}) => shallow(<StepTitle {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a Title' })

    expect(wrapper).toMatchSnapshot()
  })
})
