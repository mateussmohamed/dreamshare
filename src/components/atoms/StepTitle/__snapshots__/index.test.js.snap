// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<DreamshareTitle />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "DreamshareStepTitle-uekkjz-0",
        "isStatic": false,
        "rules": Array [
          "margin:0;",
          [Function],
          ";font-size:24px;font-weight:bold;color:#ffffff;",
        ],
      },
      "displayName": "DreamshareStepTitle",
      "render": [Function],
      "styledComponentId": "DreamshareStepTitle-uekkjz-0",
      "target": "h2",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am a Title
</StyledComponent>
`;
